# Luigi Toscano <luigi.toscano@tiscali.it>, 2016. #zanata
# Luigi Toscano <luigi.toscano@tiscali.it>, 2017. #zanata
# Giovanni Grieco <giovanni.grc96@gmail.com>, 2018. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-09-25 07:44+0200\n"
"PO-Revision-Date: 2018-04-30 08:21+0000\n"
"Last-Translator: Giovanni Grieco <giovanni.grc96@gmail.com>\n"
"Language-Team: Italian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: it\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"X-Generator: Zanata 4.6.2\n"

#. Translators: This string is only used in unit tests.
#: ../plugins/system_upgrade.py:40
msgid "the color of the sky"
msgstr "il colore del cielo"

#: ../plugins/system_upgrade.py:56
msgid "Need a --releasever greater than the current system version."
msgstr ""
"Richiesto un valore di --releasever maggiore della versione attuale del "
"sistema."

#. Translators: do not change "reboot" here
#: ../plugins/system_upgrade.py:58
msgid ""
"Download complete! Use 'dnf system-upgrade reboot' to start the upgrade.\n"
"To remove cached metadata and transaction use 'dnf system-upgrade clean'"
msgstr ""
"Download completato! Per avviare l'aggiornamento, scrivere 'dnf system-upgrade reboot'.\n"
"Per rimuovere la cache dei metadati e della transazione, scrivere 'dnf system-upgrade clean'"

#: ../plugins/system_upgrade.py:61
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr "Bisogna usare l'opzione 'download --releasever' invece di '--network'"

#: ../plugins/system_upgrade.py:236
msgid "The following boots appear to contain upgrade logs:"
msgstr ""
"I seguenti precedenti avvii del sistema sembrano contenere dei log di "
"aggiornamenti:"

#: ../plugins/system_upgrade.py:246
msgid "-- no logs were found --"
msgstr "-- nessun log è stato trovato --"

#: ../plugins/system_upgrade.py:261
msgid "Cannot find logs with this index."
msgstr "Impossibile trovare log con questo indice."

#: ../plugins/system_upgrade.py:270
#, fuzzy
msgid "Unable to match systemd journal entry"
msgstr "Non è stato possibile trovare l'inserto sul systemd journal"

#: ../plugins/system_upgrade.py:289
msgid "Prepare system for upgrade to a new release"
msgstr "Prepara il sistema per l'aggiornamento ad una nuova versione"

#: ../plugins/system_upgrade.py:299
msgid "keep installed packages if the new release's version is older"
msgstr ""
"mantiene i pacchetti installati se le versioni del nuovo rilascio sono più "
"vecchie"

#: ../plugins/system_upgrade.py:303
msgid "which logs to show"
msgstr "quali log mostrare"

#: ../plugins/system_upgrade.py:359
msgid ""
"Before you continue ensure that your system is fully upgraded by running "
"\"dnf --refresh upgrade\". Do you want to continue"
msgstr ""
"Prima di continuare assicurarsi che il sistema è completamente aggiornato "
"con \"dnf --refresh upgrade\". Procedere"

#: ../plugins/system_upgrade.py:363
msgid "Operation aborted."
msgstr "Operazione annullata."

#: ../plugins/system_upgrade.py:416
msgid "system is not ready for upgrade"
msgstr "il sistema non è pronto per gli aggiornamenti"

#: ../plugins/system_upgrade.py:418
msgid "upgrade is already scheduled"
msgstr "l'aggiornamento è già pianificato"

#: ../plugins/system_upgrade.py:424
msgid "trigger file does not exist. exiting quietly."
msgstr ""

#: ../plugins/system_upgrade.py:427
msgid "another upgrade tool is running. exiting quietly."
msgstr ""
"un altro strumento di aggiornamento è in esecuzione, uscita silenziosa."

#. Translators: do not change "reboot" here
#: ../plugins/system_upgrade.py:433
msgid "use 'dnf system-upgrade reboot' to begin the upgrade"
msgstr "usare 'dnf system-upgrade reboot' per iniziare l'aggiornamento"

#: ../plugins/system_upgrade.py:450
msgid "Rebooting to perform upgrade."
msgstr "Riavvio per effettuare l'aggiornamento."

#: ../plugins/system_upgrade.py:472 ../plugins/system_upgrade.py:478
msgid "Starting system upgrade. This will take a while."
msgstr "Avvio dell'aggiornamento del sistema. Ci vorrà un po'."

#: ../plugins/system_upgrade.py:504
#, python-format
msgid "Unable to match package: %s"
msgstr "Non è stato possibile trovare il pacchetto %s"

#: ../plugins/system_upgrade.py:509
msgid "Unable to match some of packages"
msgstr "Non è stato possibile trovare alcuni pacchetti"

#: ../plugins/system_upgrade.py:512
msgid "Cleaning up downloaded data..."
msgstr "Pulizia dei dati scaricati..."

#: ../plugins/system_upgrade.py:550
msgid "Download finished."
msgstr "Scaricamento completato."

#: ../plugins/system_upgrade.py:554 ../plugins/system_upgrade.py:555
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr "Aggiornamento completato! Pulizia e riavvio in corso..."

#: ../plugins/tracer.py:70
msgid "For more information run:"
msgstr "Per maggiori informazioni eseguire:"

#: ../plugins/tracer.py:97
msgid "Call to Tracer API ended unexpectedly:"
msgstr "La chiamata all'API di Tracer è terminata in modo inaspettato:"

#: ../plugins/tracer.py:99
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""
"Visitare https://github.com/FrostyX/tracer/issues e segnalare il problema.\n"
"Grazie"

#: ../plugins/tracer.py:101
msgid "We apologize for any inconvenience"
msgstr "Ci scusiamo per i disagi"

#: ../plugins/rpm_conf.py:67
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr "Aggiunta di '{}' all'elenco di pacchetti da far gestire a rpmconf"

#: ../plugins/rpm_conf.py:73
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr "il plugin rpmconf non verrà eseguito in modalità non-interattiva"

#: ../plugins/rpm_conf.py:86
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr ""
"viene ignorata la chiamata a sys.exit di rpmconf dovuta alla variable MERGE "
"mancante"

#: ../plugins/rpm_conf.py:90
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr ""
"viene ignorata la chiamata a sys.exit di rpmconf dovuta ad un file mancante"

#: ../plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr "connessione a snapperd non riuscita: %s"

#: ../plugins/snapper.py:55
msgid "creating pre_snapshot"
msgstr ""

#: ../plugins/snapper.py:61
#, python-format
msgid "created pre_snapshot %d"
msgstr ""

#: ../plugins/snapper.py:65
#, python-format
msgid "creating pre_snapshot failed: %s"
msgstr ""

#: ../plugins/snapper.py:71
msgid "skipping post_snapshot because creation of pre_snapshot failed"
msgstr ""

#: ../plugins/snapper.py:77
msgid "creating post_snapshot"
msgstr ""

#: ../plugins/snapper.py:82
#, python-format
msgid "created post_snapshot %d"
msgstr ""

#: ../plugins/snapper.py:86
#, python-format
msgid "creating post_snapshot failed: %s"
msgstr ""

#: ../plugins/kickstart.py:59
msgid "Install packages defined in a kickstart file on your system"
msgstr "Installa i pacchetti definiti in un file kickstart sul sistema"

#: ../plugins/kickstart.py:64
msgid "kickstart file"
msgstr "File di kickstart"

#: ../plugins/kickstart.py:82
#, python-format
msgid "file cannot be parsed: %s"
msgstr "il file non può essere analizzato: %s"

#: ../plugins/kickstart.py:99
#, python-format
msgid "No package %s available."
msgstr "Nessun pacchetto %s disponibile."

#: ../plugins/kickstart.py:104
msgid "Nothing to do."
msgstr "Niente da fare."

#: ../plugins/torproxy.py:32
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr ""
"Il plugin torproxy è stato disabilitato: impossibile connettersi alla rete "
"Tor"
